x-pack/solutions/security/plugins/security_solution/public/threat_intelligence/modules/indicators/components/table/actions_row_cell.tsx

```diff
@@ -18,24 +19,35 @@ import { INVESTIGATE_IN_TIMELINE_TEST_ID } from './test_ids';
 export const ActionsRowCell: FC<{ indicator: Indicator }> = ({ indicator }) => {
   const indicatorTableContext = useIndicatorsTableContext();
 
-  const { setExpanded, expanded } = indicatorTableContext;
+  const { setExpanded } = indicatorTableContext;
```

Comment:
Maybe this could become directy:
```tsx
const { setExpanded, expanded } = useIndicatorsTableContext();
```

Reply:
I could yeah, but this is actually being completely removed in the follow up PR ([here](https://github.com/elastic/kibana/pull/231959/files#diff-74f3fa02f720f0b1c6efd7acac5f0c166aa5ff828ed8e658bc04432e56f7cc3f)) so I don't think it matters that much 😆

Reply:
I ended up fixing it anyway as I had to rebase to avoid having the branch outdated